ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/execution/ExecutionPayloadHeader.java

```diff
@@ -102,6 +102,11 @@
         transactionsRoot);
   }
 
+  @Override
+  public boolean isDefault() {
+    return super.isDefault();
+  }
```

Comment:
I'm not sure this is right. The header of the default execution payload will return `false` from the SSZ `isDefault()` (it's not all zeros because it has the hash tree root of empty transaction list). I suspect we want to rename the method on the summary interface to `isDefaultPayload` to differentiate between the default SSZ object and the default payload.

And would be worth adding tests for the new isDefault method to make sure we are getting it right.